src/pybind/mgr/dashboard/frontend/src/styles/_carbon-defaults.scss

```diff
@@ -146,6 +147,62 @@ cd-dashboard {
 /******************************************
 Code snippet
 ******************************************/
+
 .cds--snippet {
   width: fit-content;
 }
+
+@keyframes spin {
+  to {
+    transform: rotate(360deg);
+  }
+}
+.spin {
+  transform-origin: center;
+  animation: spin 2s linear infinite;
+}
+.cds-success-color {
+  fill: theme.$support-success;
+}
+
+.cds-danger-color {
+  fill: theme.$support-error;
+}
+
+.cds-info-color {
+  fill: theme.$support-info;
+}
+
+.cds-warning-color {
+  fill: theme.$support-warning;
+}
+
+.cds--btn__icon {
+  vertical-align: text-bottom !important;
```

Comment:
I think this does not align (pun intended) with the [carbon guidelines](https://carbondesignsystem.com/elements/icons/usage/#alignment) as it states:
> When used next to text, icons should be center-aligned.

Are the icons not center-aligned by default? Do we really need this?

Reply:
yes it was not aligned...That's why  I added @Pegonzal